lighty-modules/lighty-restconf-netty-nb-community/src/main/java/io/lighty/modules/northbound/netty/restconf/community/impl/NettyRestConf.java

```diff
@@ -71,7 +71,9 @@ public NettyRestConf(final DOMDataBroker domDataBroker, final DOMRpcService domR
         this.domSchemaService = domSchemaService;
         this.inetAddress = inetAddress;
         this.httpPort = httpPort;
-        this.restconfServletContextPath = restconfServletContextPath;
+        // Remove "/" if it appears as the first character.
+        this.restconfServletContextPath = restconfServletContextPath.startsWith("/")
```

Comment:
This should be solved in configuration - for example the same way as  for JAXRS: https://github.com/PANTHEONtech/lighty/pull/2497/changes.

Reply:
Done